python/pyabacus/README.md

```diff
@@ -9,9 +9,10 @@ Python).
 Installation
 ------------
 
-- install pybind11 and scikit-build-core by `pip install pybind11 scikit-build-core`
-- clone this repository
-- `pip install ./pyabacus`
+- Create and activate a new conda env, e.g. `conda create -n myenv python=3.8 & conda activate myenv`.
+- Clone ABACUS main repository and `cd abacus-develop/python/pyabacus`.
+- Install dependencies by `pip install -r requirements.txt`.(Use `pip install -r requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple` to accelerate installation process.)
+- Build pyabacus by `pip install -v .`.
```

Comment:
Will `pip install -v .` automatically install the requirements?

Reply:
Yes, tested by @grysgreat. I think @grysgreat has followed the guide without installing `pybind` and `scikit-build-core` in your conda env manually?

Reply:
> Yes, tested by @grysgreat. 
Then is the previous step not required?
> I think @grysgreat has followed the guide without installing `pybind` and `scikit-build-core` in your conda env manually?

Are the users required to install those packages? If so, please add them to requirements.txt .

Reply:
I see what you mean. `pip install -v.` will build the packages with building dependencies by reading the `.toml` file (e.g. pybind11, scikit-build) and the `pip install -r requirements.txt` will add the dependencies for usage or test call.

Reply:
@caic99 I build pyabacus with pure gnu environment, which without installing pybind11 and scikit-build-core. And the pyabacus can properly build by following the Readme guide.

Reply:
@caic99 We can also replace the requirements.txt by adding such dependencies in `[project.optional-dependencies]` in the `.toml` file.

Reply:
> @caic99 We can also replace the requirements.txt by adding such dependencies in `[project.optional-dependencies]` in the `.toml` file.

@jieli-matrix Looks good. A unified requirement list would be better.